pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -84,15 +87,47 @@ async def run(self):
         if self.assembly_type is AssemblyTypes.STREAM:
             await self._rebase_and_build_for_stream()
         else:
+            # Check if microshift advisory is defined in assembly
+            if 'microshift' not in advisories:
```

Comment:
Previously we use `microshift: -1` to signal `prepare-release` job to create microshift advisory. Thinking we should still respect `-1` in this job. 
```suggestion
            if advisories.get(''microshift') <= 0:
```

Reply:
updated to check both